.eslintrc.js

```diff
@@ -13,6 +13,7 @@ module.exports = {
     },
     rules: {
         "no-trailing-spaces": "error",
-        "eol-last": "error"
+        "eol-last": "error",
+        semi: "error"
```

Comment:
In order to make reviews more compact, please add a comma after the last list element.